chatbot/ai-studio.yaml

```diff
@@ -1,16 +1,30 @@
-application:
-  # type: language
-  # name: chatbot
-  # description: This is a LLM chatbot application
+applications:
+  type: language
+  name: chatbot
+  description: This is a LLM chatbot application
   containers:
-    - name: chatbot-example
-      contextdir: ai-application/builds
-      #image-name: quay.io/redhat-et/chatbot:latest
+    # podman run --rm -d -p 7860:7860 quay.io/sallyom/model-services:llama2-7b-chatbot-arm
     - name: chatbot-model-arm
-      contextdir: model-services/builds/arm
+      contextdir: chatbot/model_service/builds/base
       arch: arm64
       model-service: true 
+      #image-name: quay.io/sallyom/model-services:llama2-7b-chatbot-arm
     - name: chatbot-model-x86
-      contextdir: model-services/builds/x86
+      contextdir: chatbot/model_service/builds/base
       arch: amd64
       model-service: true 
+      #image-name: quay.io/sallyom/model-services:llama2-7b-chatbot
+    - name: chatbot-model-x86-cuda
+      contextdir: chatbot/model_service/builds/cuda
+      arch: amd64
+      model-service: true 
+      #image-name: quay.io/sallyom/model-services:llama2-7b-chatbot-cuda
+    - name: chatbot-example-arm
+      contextdir: chatbot/ai_applications/builds
+      arch: arm64
+      #image-name: quay.io/sallyom/chatbot:latest-arm
+    - name: chatbot-example-x86
+      contextdir: chatbot/ai_applications/builds
+      arch: amd64
+      #image-name: quay.io/sallyom/chatbot:latest
+
```

Comment:
@MichaelClifford I used the Containerfiles from this PR to build the images above, so I'm confident they build (we'll get some GH actions going asap that will sanity-check the image builds w/ each PR). 
These are good as/of right now from this PR:
- chatbot/model_service/builds/base/Containerfile  --platform linux/amd64 && --platform linux/arm64
- summarizer/model_service/builds/base/Containerfile  --platform linux/amd64 && --platform linux/arm64
- chatbot/model_service/builds/cuda/Containerfile --platform linux/amd64
- summarizer/model_service/builds/cuda/Containerfile --platform linux/amd64
Feel free to pull the arm images down and try them.